admin/controllers/pupil.js

```diff
@@ -99,6 +98,24 @@ const postAddPupil = async (req, res, next) => {
   res.redirect(`/school/pupil-register/lastName/true?hl=${pupil._id}`)
 }
 
+const getAddMultiple = async (req, res, next) => {
+  res.locals.pageTitle = 'Add multiple pupils'
+  try {
+    req.breadcrumbs('Pupil Register', '/school/pupil-register/lastName/true')
+    req.breadcrumbs(res.locals.pageTitle)
+    res.render('school/add-multiple-pupils', {
+      breadcrumbs: req.breadcrumbs()
+    })
+  } catch (error) {
+    next(error)
+  }
+}
+
+const getMultipleTemplate = async (req, res) => {
```

Comment:
I would prefer a slightly more descriptive name: `getAddMultiplePupilsCSVTemplate` works for me for example.

Reply:
No arguments here, changed already